examples/bar.jl

```diff
@@ -35,38 +34,32 @@ function bar4()
     data = bar(;x = ["Product A", "Product B", "Product C"],
                 y = [20, 14, 23],
                 text = ["27% market share","24% market share", "19% market share"],
-                marker_color = "rgb(158,202,225)",
-                marker_opacity = 0.6,
-                line_color = "rgb(8,48,107)",
-                line_width = 1.5)
+				marker = attr(color = "rgb(158,202,225)", opacity = 0.6),
```

Comment:
actually I have that already in my vimrc. Here's my tab section. Do you know if it could be any of these other things?

"============="
" INDENTATION "
"============="
"
"" Render tabs as 4 spaces
set tabstop=4                   " Tabs are four spaces
set softtabstop=4               " Tabs counts as four spaces
set shiftwidth=4                " << >> gives you four spaces

" For those who have the sanity to use 4 spaces
set smarttab
set expandtab

set autoindent                  " Auto indent
set smartindent                 " Smart indent